x-pack/legacy/plugins/siem/public/pages/timelines/timelines_page.tsx

```diff
@@ -15,7 +15,7 @@ import { SpyRoute } from '../../utils/route/spy_routes';
 import * as i18n from './translations';
 
 const TimelinesContainer = styled.div`
-  width: 100%:
+  width: 100%;
```

Comment:
good catch here

Reply:
Yeah, nice! We used to have a linter for styled components that was nice.

Reply:
I'm happy to add it back 😄